hoodie-hadoop-mr/src/main/java/com/uber/hoodie/hadoop/realtime/AbstractRealtimeRecordReader.java

```diff
@@ -164,11 +165,26 @@ protected static String arrayWritableToString(ArrayWritable writable) {
    * columns
    */
   public static Schema generateProjectionSchema(Schema writeSchema, List<String> fieldNames) {
+    /**
+     * Avro & Presto field names seems to be case sensitive (support fields differing only in case)
+     * whereas Hive/Impala/SparkSQL(default) are case-insensitive. Spark allows this to be configurable
+     * using spark.sql.caseSensitive=true
+     *
+     * For a RT table setup with no delta-files (for a latest file-slice) -> we translate parquet schema to Avro
```

Comment:
In this case, we won't actually be using the projection schema generated right? Its only used for selecting out of logs. 

we can add a new issue around making this configurable. yes! 

nit: please remvoe the question from code comment itself :)